schemas/livePaperSection.schema.tpl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/LivePaperSection",
+  "required": ["isPartOf", "order", "sectionType", "name"],
```

Comment:
formally each item of the list should be in its own line

Reply:
done